web/src/compose_notifications.js

```diff
@@ -47,12 +48,33 @@ export function notify_above_composebox(
             link_text,
         }),
     );
-    // We pass in include_unmute_banner as false because we don't want to
-    // clear any unmute_banner associated with this same message.
-    compose_banner.clear_message_sent_banners(false);
+    // No need to clear banners that help to scroll to view or narrow to recipient
+    // while displaying the automatic new visibility policy banner.
+    if (!automatic_new_visibility_policy_banner) {
+        // We pass in include_unmute_banner as false because we don't want to
+        // clear any unmute_banner associated with this same message.
+        compose_banner.clear_message_sent_banners(false);
+    }
     compose_banner.append_compose_banner_to_banner_list($notification, $("#compose_banners"));
 }
 
+export function notify_automatic_new_visibility_policy(new_visibility_policy) {
+    let banner_text;
+    if (new_visibility_policy === user_topics.all_visibility_policies.FOLLOWED) {
+        banner_text = $t({defaultMessage: "You are now following this topic."});
+    } else {
```

Comment:
> You are now following this topic.

How about, "You are now following stream > topic." Otherwise, I think it's not super clear what "this" refers to when you are sending to a different view from the message view you're looking at.

Reply:
Ideally we'd use the appropriate stream privacy icon in the banner, but if that's going to be a whole project, we can start with a `#` and make it a follow-up.

Reply:
> we can start with a # and make it a follow-up.

Hm, that would be a follow-up as we show `#` only for the existing banners too.